CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
  ### Changed 
 * Changed `StoredQueryRepository` methods to only accept `StoredQueryQualifiedName` as arguments ([#1258](https://github.com/ehrbase/ehrbase/pull/1258))
 * Changed docker image and run integration test against it instead of the ehrbase.jar ([#1259](https://github.com/ehrbase/ehrbase/pull/1259))
+* Changed build workflow to include robot integration test coverage ([1275](https://github.com/ehrbase/ehrbase/pull/1275)) 
  ### Fixed 
```

Comment:
I do not think we need pipeline stuff in the changelog.

Reply:
removed